src/pages/faq.tsx

```diff
@@ -0,0 +1,50 @@
+import { Container, Link, Heading, Text } from "@chakra-ui/layout";
+import { Alert, AlertIcon } from "@chakra-ui/react";
+import React from "react";
+import { AiFillQuestionCircle } from "react-icons/ai";
+
+import Layout from "@/components/Layout";
+import withAuth from "@/hoc/withAuth";
+
+const FaqPage = () => {
+  return (
+    <Layout text={"よくある質問"} icon={AiFillQuestionCircle}>
+      <Container py={12} maxW="container.sm">
+        <Heading pb={3} fontSize="md">
+          Q. プルリクエストが表示されません
+        </Heading>
+        <Alert status="info">
+          <AlertIcon />
+          <Text lineHeight={6}>
+            アプリを使用するにはグループの管理者の許可が必要です
+          </Text>
+        </Alert>
+        <Text pt={3} fontSize="sm" lineHeight={5}>
+          A.
+          以下の手順を参考に、GitHubの設定を確認してください。アプリケーションのアクセスが許可されていない可能性があります。
+          <br />
+          <br />
+          1. GitHubの
+          <Link
+            onClick={() => {
+              window.open(
+                "https://github.com/settings/connections/applications/f36da358d46af3fd2862"
+              );
+            }}
+            color="teal.500"
+          >
+            設定画面
+          </Link>
```

Comment:
![must](https://img.shields.io/badge/review-must-red)
`isExternal`を指定した方が簡潔に書けるのでこちらでお願いします 🙇‍♂️ 

```suggestion
          <Link
            isExternal
            color="teal.500"
          >
            設定画面
          </Link>
```

#### 参考資料

- [Chakra UI - Link - External Link](https://chakra-ui.com/docs/navigation/link#external-link)

Reply:
`href` も必要なので、これでお願いします
```suggestion
          <Link
            href="https://github.com/settings/connections/applications/f36da358d46af3fd2862"
            isExternal
            color="teal.500"
          >
            設定画面
          </Link>
```